Source/LightboxController.swift

```diff
@@ -321,17 +255,57 @@ extension LightboxController: PageViewDelegate {
 
   func pageVewDidZoom(pageView: PageView) {
     let hidden = pageView.zoomScale != 1.0
+    let duration = hidden ? 0.0 : 1.0
```

Comment:
Don't wanna mix bool with floats, it could be not 1.0, but 1.2 for example, then conversion doesn't work.